Dockerfile

```diff
@@ -13,7 +13,7 @@ FROM --platform=${BUILDPLATFORM:-linux/amd64} docker.io/caddy:builder-alpine AS
 # see:
 # - https://github.com/caddyserver/caddy/releases
 RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} \
-      xcaddy build v2.7.0-beta.2
+      xcaddy build v2.7.5
 
 ### final image
 FROM --platform=${BUILDPLATFORM:-linux/amd64} docker.io/alpine
```

Comment:
just adding a random comment here but, if the SITE_ADDRESS is 0.0.0.0:8080 rather than :8080 the image doesn't seem to work out-of-box and it comes with couple of warnings:
```
{"level":"warn","ts":1697720505.0224447,"logger":"caddyfile","msg":"Site block has an unspecified IP address which only matches requests having that Host header; you probably want the 'bind' directive to configure the socket","address":"0.0.0.0"}

{"level":"warn","ts":1697720505.0455399,"logger":"pki.ca.local","msg":"installing root certificate (you might be prompted for password)","path":"storage:pki/authorities/local/root.crt"}

{"level":"error","ts":1697720505.0473735,"logger":"pki.ca.local","msg":"failed to install root certificate","error":"failed to execute tee: exit status 1","certificate_file":"storage:pki/authorities/local/root.crt"}
```